common/recommender/Baseline.js

```diff
@@ -2,6 +2,7 @@
 
 const URL = require("../../lib/vendor.bundle.js").urlParse;
 const {INFINITE_SCROLL_THRESHOLD} = require("../constants");
+const COEFFICIENTS = [0.4, 0.7, 0.1, -0.4, -0.2, -0.1];
```

Comment:
@pdehaan you can load the package.json and read the values. Not sure if that's a good practice™®